### ISSUE TITLE
Create and document process to localize translationStudio 

### ISSUE THREAD

**Issue by mondele** [unfoldingWord-dev/ts-desktop#1195]
Now that translationStudio is in maintenance mode, it seems that we should open up the possibilities of localization so that our MTTs can work with the program in the most accessible idiom possible.

**Comment by mondele** [comment]
I would like to bump this issue, since this is an important missing feature.

**Comment by jag3773** [comment]
@neutrinog Can you please post the link to the crowd site that people can go to fill out the localization?  If Android and Desktop have different strings, can you post both?

**Comment by neutrinog** [comment]
This is the project for tS Android https://crowdin.com/project/translation-studio. We never got to the point where we were localizing strings for the desktop app. So there's no infrastructure within the application to receive the localization.

Also, we'd need to update the Android project on Crowdin with the latest English strings before we started working on this again. It's not hard I just need to upload a couple files. I can do so tomorrow.

**Comment by mondele** [comment]
So, at this point is there no path forward to localize translationStudio Desktop?

**Comment by neutrinog** [comment]
Not without further development.

**Comment by neutrinog** [comment]
I just updated the localization files for translationStudio Android on Crowdin and am running a pre-translation process using the Google translate api. Feel free to have people start translating!

**Comment by mondele** [comment]
Can we change this issue into a request for additional development to allow localization of the UI, or should I submit a different issue?

**Comment by jag3773** [comment]
@mondele Yes, consider this a feature request for additional development to support a localizable UI.

**Comment by neutrinog** [comment]
Dumping my notes for localizing the UI so I don't lose them.

- [ ] localize the 150-200 strings in the app.
- [ ] improve localization library already inside tS.
  - [ ] support nested json structure in locale strings
- [ ] build ui for changing the locale (in the settings screen)
- [ ] persist locale settings to disk
- [ ] load system locale by default
